inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -482,12 +484,20 @@ QueryNetworkResult Engine::QueryNetwork(const CNNNetwork& network, const std::ma
                 }
             }
         }
-
-        for (auto&& layerName : supported) {
-            if (!InferenceEngine::details::contains(unsupported, layerName)) {
-                res.supportedLayersMap.emplace(layerName, GetName());
+        for (auto&& node : function->get_ops()) {
```

Comment:
Looks like we can move the body of this cycle to previous one. Am I not right?

Reply:
merged the cycles.